libs/sdk-ui-ext/src/internal/dashboardEmbedding/KpiView/KpiRenderer.tsx

```diff
@@ -72,7 +74,7 @@ export const KpiRenderer: React.FC<IKpiRendererProps> = ({
                 </div>
                 <ResponsiveText>
                     <div className="headline-value-wrapper" onClick={onPrimaryValueClick}>
-                        {primaryValue?.isDrillable ? (
+                        {primaryValue?.isDrillable && !disableDrillUnderline ? (
```

Comment:
This is not so simple, even with `disableDrillUnderline` it should be rendered as link, just without the "headline-link-style-underline" class (see TODO on line 30).